addons/hr/hr.py

```diff
@@ -316,10 +316,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user_obj = self.pool.get('res.users').browse(cr, uid, user_id, context=context)
```

Comment:
I agree about image, as i understand from debugging there are 3 type of images - native, medium and small, however Employee had place only for medium image, so I wasn't sure if I can assign native image! didn't want to break anything